modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc

```diff
@@ -358,6 +358,12 @@ NAMESPACE                              NAME            CSV
 openshift-logging                      install-6khtw   cluster-logging.5.3.3-4             Manual     true <1>
 ----
 <1> The install plans have their `Approval` field set to `Manual` and their `Approved` field changes from `true` to `false` after {cgu-operator} approves the install plan.
```

Comment:
should be false to true

Reply:
thanks. Updated and added a note to the JIRA.